src/status_im/constants.cljs

```diff
@@ -10,8 +10,6 @@
 (def content-type-text "text/plain")
 (def content-type-sticker "sticker")
 (def content-type-status "status")
-(def content-type-command "command")
```

Comment:
@cammellos i've seen you make changes here as well. jfyi

Reply:
yes, thanks, @yenda mentioned this as well